pkg/vm/engine/aoe/storage/wal/shard/shard_test.go

```diff
@@ -121,8 +196,7 @@ func TestShardProxy(t *testing.T) {
 			ic := ii % cnt
 			consumers[ic].consume(index)
 			ii++
-			idx := &(*index)
-			shardProxy.LogIndex(idx)
+			shardProxy.LogIndex(index)
```

Comment:
***LogIndex*** doesn't guarantee it won't rewrite the incoming **index**. For safety, make a copy and send a pointer to the copy.